src/main/java/net/mcreator/ui/dialogs/tools/WoodPackMakerTool.java

```diff
@@ -0,0 +1,447 @@
+		fenceRecipe.recipeSlots[4] = new MItemBlock(workspace, "Items.STICK");
+		fenceRecipe.recipeSlots[5] = new MItemBlock(workspace, "CUSTOM:" + planksBlock.getModElement().getName());
+		fenceRecipe.recipeSlots[6] = new MItemBlock(workspace, "CUSTOM:" + planksBlock.getModElement().getName());
+		fenceRecipe.recipeSlots[7] = new MItemBlock(workspace, "Items.STICK");
+		fenceRecipe.recipeSlots[8] = new MItemBlock(workspace, "CUSTOM:" + planksBlock.getModElement().getName());
+		fenceRecipe.recipeReturnStack = new MItemBlock(workspace, "CUSTOM:" + name + "Fence");
+		fenceRecipe.recipeRetstackSize = 3;
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(fenceRecipe);
+		mcreator.getWorkspace().addModElement(fenceRecipe.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(fenceRecipe);
+		mcreator.getWorkspace().getModElementManager().storeModElement(fenceRecipe);
+
+		Recipe fenceGateRecipe = (Recipe) ModElementTypeRegistry.REGISTRY.get(ModElementType.RECIPE)
+				.getModElement(mcreator, new ModElement(workspace, name + "FenceGateRecipe", ModElementType.RECIPE),
+						false).getElementFromGUI();
+		fenceGateRecipe.recipeSlots[3] = new MItemBlock(workspace, "Items.STICK");
+		fenceGateRecipe.recipeSlots[4] = new MItemBlock(workspace, "CUSTOM:" + planksBlock.getModElement().getName());
+		fenceGateRecipe.recipeSlots[5] = new MItemBlock(workspace, "Items.STICK");
+		fenceGateRecipe.recipeSlots[6] = new MItemBlock(workspace, "Items.STICK");
+		fenceGateRecipe.recipeSlots[7] = new MItemBlock(workspace, "CUSTOM:" + planksBlock.getModElement().getName());
+		fenceGateRecipe.recipeSlots[8] = new MItemBlock(workspace, "Items.STICK");
+		fenceGateRecipe.recipeReturnStack = new MItemBlock(workspace, "CUSTOM:" + name + "FenceGate");
+		fenceGateRecipe.recipeRetstackSize = 1;
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(fenceGateRecipe);
+		mcreator.getWorkspace().addModElement(fenceGateRecipe.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(fenceGateRecipe);
+		mcreator.getWorkspace().getModElementManager().storeModElement(fenceGateRecipe);
+	}
+
+	public static BasicAction getAction(ActionRegistry actionRegistry) {
+		return new BasicAction(actionRegistry, "Create wood pack...", e -> open(actionRegistry.getMCreator())) {
+			@Override public boolean isEnabled() {
+				GeneratorConfiguration gc = actionRegistry.getMCreator().getWorkspace().getGenerator()
+						.getGeneratorConfiguration();
+				return gc.getGeneratorStats().getModElementTypeCoverageInfo().get(ModElementType.RECIPE)
+						!= GeneratorStats.CoverageStatus.NONE
+						&& gc.getGeneratorStats().getModElementTypeCoverageInfo().get(ModElementType.BLOCK)
+						!= GeneratorStats.CoverageStatus.NONE
+						&& gc.getGeneratorStats().getModElementTypeCoverageInfo().get(ModElementType.TAG)
+						!= GeneratorStats.CoverageStatus.NONE;
+			}
+		}.setIcon(UIRES.get("16px.armorpack"));

```

Comment:
Make a new icon, or use no icon.

Reply:
Don't worry, he said that he'll change the icon. :)

Reply:
My bad, did not notice :D I suggest adding placeholder than and fixing name, and then Matej will work on it ;)